autosklearn/metalearning/metalearning/meta_base.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, configuration_space, aslib_directory, logger):
         for algorithm_id in self.configurations:
             configuration = self.configurations[algorithm_id]
             try:
+                # ToDo check if all stored configurations are actually readable
```

Comment:
What does this mean?

Reply:
I have noticed that some configurations cant be loaded because in the referring configurations.csv the value 999 instead of an float < 1.0 is stored i think that is the bug which makes waringings like:

[WARNING] [2021-11-04 10:16:34,533:Client-AutoMLSMBO(1)::e6ac0eb2-3d4f-11ec-9631-98bb1e1dd0b6] Configuration 555 not found

if you want to reconstruct this the problem i found you can comment out the try / except statement and just add the line where the configurations loaded without try statement then you will get an error like:

  File "/home/lukas/Python_Projects/NewBOWDev/auto-sklearn/autosklearn/metalearning/metalearning/meta_base.py", line 52, in __init__
    (Configuration(configuration_space, values=configuration))
  File "ConfigSpace/configuration_space.pyx", line 1412, in ConfigSpace.configuration_space.Configuration.__init__
ValueError: Trying to set illegal value '999' (type '<class 'int'>') for hyperparameter 'classifier:mlp:beta_2, Type: Constant, Value: 0.999' (default-value has type '<class 'float'>').

Reply:
That's an interesting bug to have discovered, we should probably make this as an issue as it's come up a few times in the past.

Reply:
I cannot reproduce this.